test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -982,15 +982,15 @@ private async Task HelloWorldOrchestration_Activity_Main_Logic(string taskHubNam
 
                     if (showHistoryOutput)
                     {
-                        Assert.Null(status.History[0]["Input"]);

```

Comment:
Is this a breaking change? Were we previously returning the orchestration history when `showHistoryOutput` was true? If so, we should not change this behavior or else existing customers may be impacted.

Tip: we always have to be extremely careful about accidentally introducing breaking changes to the code. If a change requires us to modify a unit test like this, that's a good hint that we may be accidentally making a breaking change.

Reply:
Oh, sorry, I think I was wrong. This is actually *not* a breaking change because the input, which was previously null, is now non-null. I looked at the diff wrong. I thought it was the other way around.